middleware/index.js

```diff
@@ -67,7 +67,7 @@ const authMiddleware = () => {
 			ALLOWED_DOMAINS,
 			OAUTH_CLIENT_ID,
 			OAUTH_CLIENT_SECRET,
-			GIT_HOSTNAME = "github.com",
+			GIT_HOSTNAME = provider === "github" ? "github.com" : "gitlab.com",
```

Comment:
You don't need to change this, but it occurred to me that a simplification would be to do the following:

```suggestion
			GIT_HOSTNAME = `${provider}.com`,
```

That being said it may not be applicable for other providers in the future.